setup.py

```diff
@@ -19,7 +19,7 @@
     author_email='russell@keith-magee.com',
     url=data['homepage'],
     packages=find_packages(exclude=['docs', 'tests']),
-    python_requires='>=3.4, <=3.5',
+    python_requires='>=3.4, <=3.6',
```

Comment:
If we put <=3.6, I think we will get in to the situation where 3.6.1 is not allowed. What if instead we say <3.7 instead of <=3.6?

Reply:
That is a good idea